### DISCUSSION TITLE
🔙 Page jumps back to top 🔝

### DISCUSSION THREAD

**Discussion by hasan-py** [community/community#13208] [Other Feature Feedback, Questions, & Ideas]
When I visit some page from the feed and then come back to the feed it jumps back to the top. It should be where I was clicking or which card I was a click.

**Comment by jfullstackdev** [comment]
yes much like Twitter,

but just to update this also

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

much like minimizing what you want to see
